node/sifrr-server/config.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+  website: sifrr.github.io/sifrr/#/packages/server/sifrr-server/
```

Comment:
seems to be the wrong URL, **SHOULD** be `website: sifrr.github.io/sifrr/#/./packages/server/sifrr-server/`

Reply:
works for me 🤔